### PR TITLE
fix: Add validations for aliases

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Currently aliases for helm-requirements is failing when configured during nested json validation.

```
INFO: Repository has invalid config
{
  "error": {
    "configFile": ".github/renovate.json",
    "validationError": "The renovate configuration file contains some invalid settings",
    "validationMessage": "Invalid configuration option: helm-requirements.aliases.stable"
  }
```

Closes https://github.com/renovatebot/config-help/issues/679

### CODE REVIEW

**Comment by adusumillipraveen** [comment]
@rarkins any thoughts on this PR ? I am not sure if this is the right fix, but happy to contribute if you can guide me.

**Comment by rarkins** [comment]
@adusumillipraveen we don't actually do the validation you're seeing using a JSON validator - it's an internal tool with a bit more config awareness. So unfortunately I think the solution is instead to add `aliases` awareness to https://github.com/renovatebot/renovate/blob/master/lib/config/validation.ts

We should check:
- aliases is an object
- The object is one deep
- values of each field are valid URIs

**Comment by adusumillipraveen** [comment]
> @adusumillipraveen we don't actually do the validation you're seeing using a JSON validator - it's an internal tool with a bit more config awareness. So unfortunately I think the solution is instead to add `aliases` awareness to https://github.com/renovatebot/renovate/blob/master/lib/config/validation.ts
> 
> We should check:
> 
> * aliases is an object
> * The object is one deep
> * values of each field are valid URIs

Thanks @rarkins , i pushed a commit to add specific validations for aliases along with some tests. Please review.

File: lib/config/validation.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import is from '@sindresorhus/is';
+import isUrl from 'is-url';
```
Comment (rarkins) [inline lib/config/validation.ts:2]: Can [is.urlString](https://github.com/sindresorhus/is#urlstringvalue) work instead?

**Review by rarkins — CHANGES_REQUESTED** [review]
Please check if we can reuse the existing `is` module for url validation

File: lib/config/validation.ts (hunk 1 above)
Comment (adusumillipraveen) [reply lib/config/validation.ts]: Done, thanks.

**Review by rarkins — APPROVED** [review]

**Pull request merged**